apps/files/js/app.js

```diff
@@ -166,6 +169,28 @@
 			}
 		},
 
+		/**
+		 * Set the currently active file list
+		 *
+		 * Due to the file list implementations being registered after clicking the
+		 * navigation item for the first time, OCA.Files.App is not aware of those until
+		 * they have initialized themselves. Therefore the files list needs to call this
+		 * method manually
+		 *
+		 * @param {OCA.Files.FileList} newFileList
+		 */
+		updateCurrentFileList: function(newFileList) {
+			this.currentFileList = newFileList;
+		},
+
+		/**
+		 * Return the currently active file list
+		 * @return {OCA.Files.FileList}
```

Comment:
could also be undefined/null?
```suggestion
		 * @return {?OCA.Files.FileList}
```

Reply:
Should not happen since it is set in the initialize function, but added just to be sure.